src/net_processing.cpp

```diff
@@ -568,13 +568,13 @@ struct CNodeState {
         bool m_protect;
     };
 
-    ChainSyncTimeoutState m_chain_sync;
+    ChainSyncTimeoutState m_chain_sync{0, nullptr, false, false};
```

Comment:
Any reason not to add these default initializers to the `ChainSyncTimeoutState` members themselves?

Reply:
Done in #21424